src/modules/map/components/cesium/constants.js

```diff
@@ -0,0 +1,12 @@
+import { Rectangle } from 'cesium'
+
+export const SWITZERLAND_BOUNDS = [5.140242, 45.398181, 11.47757, 48.230651]
+
+export const SWITZERLAND_RECTANGLE = Rectangle.fromDegrees(...SWITZERLAND_BOUNDS)
```

Comment:
I've seen that there is a `.fromCartesianArray`, it might be a good candidate to use our `LV95_EXTENT` constant directly here